README.md

```diff
@@ -3,7 +3,7 @@
 Exchange Calendar is an add-on for [Thunderbird](https://mozilla.org/thunderbird)
 and [Seamonkey](https://www.seamonkey-project.org/).
 
-It extends the [Lightning](https://mozilla.org/calendar) add-on to provide
+It extends the [Lightning](https://addons.mozilla.org/de/thunderbird/addon/lightning/) add-on to provide
```

Comment:
should link to https://addons.mozilla.org/thunderbird/addon/lightning/ (without language code) because Mozilla reads preferred languages from browser settings

Reply:
Indeed, that's a good suggestion, thanks.

@smehrbrodt could you update the PR with this link please ?